lib/services/matchers.go

```diff
@@ -187,6 +200,21 @@ func MatchResourceByFilters(resource types.ResourceWithLabels, filter MatchResou
 	return match, nil
 }
 
+func hasTeleportInternalLabel(resource types.ResourceWithLabels) bool {
+	if resource == nil {
+		return false
+	}
+
+	labels := resource.GetAllLabels()
+	for name := range labels {
+		if strings.Contains(name, "teleport.internal") {
```

Comment:
`teleport.internal/` is a prefix, so you should use `strings.HasPrefix` instead.